src/k4a_ros_device.cpp

```diff
@@ -874,12 +874,16 @@ void K4AROSDevice::framePublisherThread()
   calibration_data_.getRgbCameraInfo(depth_rect_camera_info);
   calibration_data_.getDepthCameraInfo(ir_raw_camera_info);
 
+  const std::chrono::milliseconds firstFrameWaitTime = std::chrono::milliseconds(4 * 1000);
+  const std::chrono::milliseconds regularFrameWaitTime = std::chrono::milliseconds(1000 * 5 / params_.fps);
```

Comment:
[nit] could name these kFirstFrameWaitTime and kRegularFrameWaitTime to show they're constants

Reply:
Keeping the same for consistency with the other const in the file.